test/e2e/config/nutanix.yaml

```diff
@@ -79,6 +79,8 @@ providers:
         files:
           - sourcePath: "../../../metadata.yaml"
           - sourcePath: "../data/infrastructure-nutanix/v1beta1/cluster-template.yaml"
+          - sourcePath: "../data/infrastructure-nutanix/v1beta1/cluster-template-no-secret.yaml"
```

Comment:
normally cluster-templates correspond to flavors. here we are using it for testing. just wondering if we can do the same without creating a flavor but thru kustomize. pls see the crs.yaml